### PR TITLE
changes in matrix os > basic dashboard and installation

### PR DESCRIPTION
**Pull request by heitorgo1**
fix broken link to Hello World example in installation page;
fix config.yaml format in basic dashboard example;

### CODE REVIEW

File: docs/matrix-os/examples/dash-create.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ widgets:
 `message` is the widget name to add to the `screens` layout definition.
 ```yaml
 screens:
-  - message
```
Comment (eighteyes) [inline docs/matrix-os/examples/dash-create.md]: no, this is intentional

File: docs/matrix-os/examples/dash-create.md [hunk 2]
```diff
@@ -71,14 +71,14 @@ widgets:
     label: from device
 
 screens:
-  - message
```
Comment (eighteyes) [inline docs/matrix-os/examples/dash-create.md]: this is intentional

Comment (heitorgo1) [reply docs/matrix-os/examples/dash-create.md]: Sean, everytime I tried without an additional dash the matrix OS said there was a formatting error. Should we keep only one dash anyway?

Comment (eighteyes) [reply docs/matrix-os/examples/dash-create.md]: Something happened. it used to be two dashes. :(

**Pull request merged**